class-5/prototypal_Inheritance.js

```diff
@@ -0,0 +1,25 @@
+// function Dog(name, breed, color) {
```

Comment:
You can discard commenting this code block. Here we can define this functional constructor as `DogFunc` and for class constructor as `DogClass`.